examples/stable_diffusion_v2/requirements.txt

```diff
@@ -9,3 +9,7 @@ imagesize
 toolz
 pillow==9.1.1
 tqdm
+imageio==2.31.1
+webdataset==0.2.48
+addict==2.4.0
+yapf==0.40.1
```

Comment:
don't add dependencies that are not used in the project.

Reply:
fixed